body/stretch_body/pimu.py

```diff
@@ -351,41 +335,6 @@ def unpack_board_info(self,s):
             return sidx
 
 
-    def unpack_status(self,s):
-        with self.lock:
-            sidx=0
-            sidx +=self.imu.unpack_status((s[sidx:]))
-            self.status['voltage']=self.get_voltage(unpack_float_t(s[sidx:]));sidx+=4
-            self.status['current'] = self.get_current(unpack_float_t(s[sidx:]));sidx+=4
-            self.status['temp'] = self.get_temp(unpack_float_t(s[sidx:]));sidx+=4
-
-            for i in range(4):
-                self.status['cliff_range'][i]=unpack_float_t(s[sidx:])
-                sidx+=4
-
-            self.status['state'] = unpack_uint32_t(s[sidx:])
-            sidx += 4
-
-            self.status['at_cliff']=[]
-            self.status['at_cliff'].append((self.status['state'] & STATE_AT_CLIFF_0) != 0)
-            self.status['at_cliff'].append((self.status['state'] & STATE_AT_CLIFF_1) != 0)
-            self.status['at_cliff'].append((self.status['state'] & STATE_AT_CLIFF_2) != 0)
-            self.status['at_cliff'].append((self.status['state'] & STATE_AT_CLIFF_3) != 0)
-            self.status['runstop_event'] = (self.status['state'] & STATE_RUNSTOP_EVENT) != 0
-            self.status['cliff_event'] = (self.status['state'] & STATE_CLIFF_EVENT) != 0
-            self.status['fan_on'] = (self.status['state'] & STATE_FAN_ON) != 0
-            self.status['buzzer_on'] = (self.status['state'] & STATE_BUZZER_ON) != 0
-            self.status['low_voltage_alert'] = (self.status['state'] & STATE_LOW_VOLTAGE_ALERT) != 0
-            self.status['high_current_alert'] = (self.status['state'] & STATE_HIGH_CURRENT_ALERT) != 0
-            self.status['over_tilt_alert'] = (self.status['state'] & STATE_OVER_TILT_ALERT) != 0
-            self.status['timestamp'] = self.timestamp.set(unpack_uint64_t(s[sidx:])); sidx += 8
-            self.imu.status['timestamp'] = self.status['timestamp']
-            timestamp_line_sync = unpack_uint64_t(s[sidx:]); sidx += 8
-            self.status['bump_event_cnt'] = unpack_uint16_t(s[sidx:]);sidx += 2
-            self.status['debug'] = unpack_float_t(s[sidx:]); sidx += 4
-            self.status['cpu_temp']=self.get_cpu_temp()
-            return sidx
-
     def pack_config(self,s,sidx):
```

Comment:
`pack_config` differs in P0 and P1: P1 introduces the "enable_sync_mode" config to PIMU. `pack_config` should move to the Protocol_Px classes.

Reply:
The 'new' P1 under bugfix/remove_sync_timestamps doesn't include enable_sync_mode so leaving pack_config out of Px for now.